docs/reference/content/reference/observables.md

```diff
@@ -0,0 +1,126 @@
++++
+date = "2015-05-14T08:01:00-00:00"
+title = "Observables"
+[menu.main]
+  parent = "Reference"
+  identifier = "Observables"
+  weight = 75
+  pre = "<i class='fa'></i>"
++++
+
+## Observables
+
+The MongoDB Scala Driver is an asynchronous and non blocking driver. Using the `Observable` model asynchronous events become simple, composable operations, freed from the complexity of nested callbacks.  
+
+For asynchronous operations there are three interfaces [`Observable`]({{< apiref "org.mongodb.scala.Observable" >}}), [`Subscription`]({{< apiref "org.mongodb.scala.Subscription" >}}) and [`Observer`]({{< apiref "org.mongodb.scala.Observer" >}}).
+
+{{% note %}}
+The interfaces are the similar to `Publisher`, `Subscription` and `Subscriber` interfaces from the [reactive streams](http://www.reactive-streams.org/) JVM implementation.  However, we prefer the name `Observerable` to `Publisher` and `Observer` to `Subscriber` for readability purposes.
+{{% /note %}}
+
+## Observable
+The [`Observable`]({{< apiref "org.mongodb.scala.Observable" >}}) represents a MongoDB operation which emits its results to the `Observer` based on demand requested by the `Subscription` to the `Observable`. 
+
+## Subscription
+
+A [`Subscription`]({{< apiref "org.mongodb.scala.Subscription" >}}) represents a one-to-one lifecycle of an `Observer` subscribing to an `Observable`.  A `Subscription` to an `Observable` can only be used by a single `Observer`.  The purpose of a `Subscription` is to control demand and to allow unsubscribing from the `Observable`.
+
+## Observer
+
+An [`Observer`]({{< apiref "org.mongodb.scala.Observer" >}}) provides the mechanism for receiving push-based notifications from the
+`Observable`.  Demand for these events is signalled by its `Subscription`.
+
+On subscription to an `Observable[TResult]` the `Observer` will be passed the `Subscription` via the 
+`onSubscribe(subscription: Subscription)`. Demand for results is signaled via the `Subscription` and any results are passed to the 
+`onNext(result: TResult)` method.  If there is an error for any reason the `onError(e: Throwable)` will be 
+called and no more events passed to the `Observer`. Alternatively, when the `Observer` has consumed all the results from the `Observable` 
+the `onComplete()` method is called.
```

Comment:
"will be called" instead of "is called", to match tense with the previous part of the sentence.